cmd/contour/serve.go

```diff
@@ -160,6 +163,12 @@ func doServe(log logrus.FieldLogger, ctx *serveContext) error {
 		}
 	}
 
+	// Before we can build out the event handler, we need to initialize the converter we'll
+	// use to convert from Unstructured. Thanks to kubebuilder types from service-apis, this now can
+	// return an error.
+
+	converter, err := k8s.NewUnstructuredConverter()
+	check(err)
```

Comment:
Initialization of the service-apis failing now breaks Contour from working altogether.  Should we change the `check(err)` to just log the error? Or only use `check` if the apis are enabled?

I'm just worried that this code might somehow break users who don't buy into the experimental flag.

Reply:
If we make the initialisation of the converter check for the flag, then it shouldn't be able to affect anyone else, thanks for catching that in `NewUnstructuredConverter()`.